collections/_posts/2022-11-14-DeformableConvolutionalNetworks.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: review
+title: "Deformable Convolutional Networks"
+tags:  Convolution, recepteive field, atrou
+author: "Nolann Lainé"
+cite:
+    authors: "Jifeng Dai, Haozhi Qi, Yuwen Xiong, Yi Li, Guodong Zhang, Han Hu, Yichen Wei"
+    title:   "Deformable Convolutional Networks"
+    venue:   "Microsoft Research Asia"
+pdf: "https://arxiv.org/pdf/1703.06211v3.pdf"
+---
+
+# Notes
+
+* Code is available here: [repo]( https://github.com/msracver/Deformable-ConvNets)
+
+# Highlights
+
+* The objective of this paper is to propose a method to adapt CNN to different geometries.
+* The innovation comes from the addition of layers that learn the "stride" in order to have filters on a non-regular grid.
+* The proposed method is interesting for classification, segmentation and object tracking tasks.
+&nbsp;
+
+
+# Introduction 
+
+For a neural network, geometric transformation, viewpoint and partial deformation are difficult to learn. Most of the time, to generalise an architecture during learning, known geometric deformations such as scaling, rotation or shearing are applied. 
+However, there is no mechanism to help the model learn the geometric variations. The data augmentation is therefore limited by the user's knowledge. The proposed mechanism is end-to-end, adaptable to each architecture containing CNNs and allows CNNs to learn more complex deformations.
```

Comment:
> The data augmentation is therefore limited by the user's knowledge.

It's not quite clear to me what the _user's knowledge_ is supposed to be here. Do you mean the user's expertise to implement possibly more complex geometrical transformation? I think that sentence deserves to be explained a bit more.